src/VS.Web.CG.Mvc/Identity/IdentityGeneratorTemplateModel2.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.VisualStudio.Web.CodeGenerators.Mvc.Identity
+{
+    public class IdentityGeneratorTemplateModel2 : IdentityGeneratorTemplateModel
```

Comment:
We can't modify a public class in a minor release, it would break people trying to extend scaffolding. (I talk about this a little more in another comment).